### PR TITLE
Pin dependencies

### PR DESCRIPTION
**Pull request by jkimbo**
Improves on https://github.com/graphql-python/sanic-graphql/pull/17 and fixes the tests

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,17 +2,18 @@ language: python
...
 matrix:
   include:
-  - python: '3.5'
-    env: TOX_ENV=py35,flake8
+  - python: '3.7'
+    env: TOXENV=flake8
   - python: '3.6'
-    env: TOX_ENV=py36
+  - python: '3.7'
+  - python: '3.8'
```
Comment (KingDarBoja) [inline .travis.yml]: These can be defined at the `language` section of travis. See [graphql-server-core#35 travis](https://github.com/graphql-python/graphql-server-core/blob/d8b98b7342bc4a451ffb324082f4f8dd3b9397e1/.travis.yml) file.

File: setup.py [hunk 2]
```diff
@@ -1,34 +1,37 @@
...
-    long_description=open('README.rst').read(),
-    url='https://github.com/graphql-python/sanic-graphql',
-    download_url='https://github.com/graphql-python/sanic-graphql/releases',
-    author='Sergey Porivaev',
-    author_email='porivaevs@gmail.com',
-    license='MIT',
+    name="Sanic-GraphQL",
+    version="1.1.0",
+    description="Adds GraphQL support to your Sanic application",
+    long_description=open("README.rst").read(),
```
Comment (KingDarBoja) [inline setup.py]: Would be good to provide encoding as `utf-8` in case any emoji is being used on the readme body.

**Comment by jkimbo** [comment]
@KingDarBoja ready for review again.

**Review by KingDarBoja — APPROVED** [review]
LGTM 🚀

**Pull request merged**